server/kv.go

```diff
@@ -369,7 +367,7 @@ func (store store) EnsureRSAKey() (rsaKey *rsa.PrivateKey, returnErr error) {
 
 	if rsaKey == nil {
 		var newRSAKey *rsa.PrivateKey
-		newRSAKey, err = rsa.GenerateKey(rand.Reader, 1024)
+		newRSAKey, err = rsa.GenerateKey(rand.Reader, 2048)
```

Comment:
Please doublecheck this change has no effects

Reply:
While I don't think this has an immediate effect, I would like to keep the key length consistent between all instances. Would you please revert the changes?

Reply:
Done :)